src/php/utils/docker/docker-php-dev-mode

```diff
@@ -29,7 +29,15 @@ case "$1" in
 			exit 1
 		fi
 
-		pecl install xdebug-stable || pecl install xdebug-2.9.8
+		# If php version >= 8.0 install the right version of xdebug and linux headers
+		php_version=$(php -r 'echo PHP_VERSION;' | cut -c1-3 | tr -d '\n')
+		if [ "$(echo "$php_version >= 8.0" | bc)" = 1 ]; then
+			apk add --update linux-headers
+			pecl install xdebug
```

Comment:
Would be better to keep `xdebug-stable` here? I'm not sure if the `xdebug` includes beta versions too.

Reply:
ok, let me edit it and see what happens